scripts/install_ebpf.psm1

```diff
@@ -296,11 +296,10 @@ function Install-eBPFComponents
     }
 
 
-    # Copy ebpfapi.dll from installation directory to current directory.
-    # This ensures latest version of ebpfapi is used by the tests.
-    $ebpfApiDllPath = Join-Path $env:ProgramFiles "ebpf-for-windows\ebpfapi.dll"
-    Write-Log("Copying ebpfapi.dll from '$ebpfApiDllPath' to '$pwd'...")
-    Copy-Item -Path $ebpfApiDllPath -Destination $pwd -Force
+    # Refresh Path so EbpfApi.dll can be found.
```

Comment:
The ebpfapi.dll is placed in ProgramFiles. In your changes, I do not see the dll being fetched from the ProgramFiles. Can you please help clarify?

Reply:
The installer copies the dll over and adds the eBPF directory to the system Path, but that doesn't update Path in the current PowerShell session.

This change re-loads the system Path after running the installer so the eBPF Program Files directory gets included and the dll found.

Reply:
Thanks  @mikeagun for the clarification. 
1. Does it require admin privilege? 
2. Does it also remove the system path at uninstall?

Reply:
1. No additional privileges needed -- this doesn't modify the system Path, just loads the changes the installer previously made.
2. I didn't add anything to modify Path in the current shell session at uninstall time, but the uninstaller already removes the eBPF directory from the system Path.